andromedocs-theme/src/components/layout/layout.tsx

```diff
@@ -1,17 +1,30 @@
 import React from "react";
 
 import Header from "../header";
+import Sidebar from "../sidebar";
 import Footer from "../footer";
 import { Box, Flex, Text } from "theme-ui";
 
 const Layout = ({ children, title }) => (
   <Flex sx={{ flexDirection: "column", minHeight: "100vh" }}>
     <Header />
-    <Text as="h1">{title}</Text>
-    <Box as="main" sx={{ flex: "1 1 0%" }}>
-      {children}
-    </Box>
-    <Footer />
+    <Flex>
+      <Sidebar />
+      <Flex sx={{ minHeight: "calc(100vh - 80px)", flexDirection: "column", flex: "0 0 85%" }}>
```

Comment:
After this PR we can start working on placing these values in our ThemeUI theme :sparkles: